src/components/Media/index.js

```diff
@@ -7,71 +7,102 @@
 import React, { Component } from "react";
 import Proptypes from "prop-types";
 import "./Media.css";
-function Media() {
-  return <div>{<InnerFunc typeGet="nested" />}</div>;
-}
-
-function InnerFunc(props) {
-  if (props.typeGet === "linear") {
-    return (
-      <div className="Media">
-        <div>
-          <img
-            height={64}
-            width={64}
-            src="https://tinyurl.com/yylsuh97"
-            alt="Generic placeholder"
-          />
-        </div>
-        <div>
-          <h3>Media Heading</h3>
-          <p>{props.children}</p>
-        </div>
-      </div>
-    );
-  } else if (props.typeGet === "nested") {
-    return (
-      <div>
-        <div className="Media">
-          <div>
-            <img
-              height={64}
-              width={64}
-              src="https://tinyurl.com/yylsuh97"
-              alt="Generic placeholder"
-            />
-          </div>
+      function Media(props) {
+        const {typeGet, image} = props;
```

Comment:
great destructuring. 
we can even do it directly inside the params of the function
Eg:
```js
function Media({typeGet, image}){}
```
like this